plugins/net_plugin/net_plugin.cpp

```diff
@@ -2373,8 +2378,7 @@ namespace eosio {
                   c->send_handshake();
                }
             } else {
-               fc_elog( logger, "connection failed to ${host}:${port} ${error}",
-                        ("host", endpoint.address().to_string())("port", endpoint.port())( "error", err.message()));
+               fc_elog( logger, "connection failed to ${a}, ${error}", ("a", c->peer_address())( "error", err.message()));
```

Comment:
wouldn't `"connection to ${a} failed, ..."` be better english?

Reply:
Maybe. I would say "connection failed to" is easier to grep.

Reply:
odd justification, but OK :-). If we want to optimize for grep, maybe we should add random typos as well.

Reply:
lol, I have found random typos in the past to be helpful actually. I don't have a strong opinion about this at all. Happy to change it if you like.

Reply:
No if you like it better like that, it is fine, you certainly look  at logs much more than me.